sequencing-server/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sequencing-server",
   "version": "1.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
@goetzrrGit What version of Node/NPM are you using? The lock file should not have this many updates with the change you made to package.json. Make sure you are using node LTS (which comes with NPM 8.x). The lockfile version should remain on `2`.

Reply:
This was generated by` gradlew assemble `which is node 18.13 using npm that is bundled in 18.13